app-accessibility/edbrowse/edbrowse-3.8.2.1-r2.ebuild

```diff
@@ -12,7 +12,7 @@ QUICKJS_P="quickjs-${QUICKJS_SHORT}"
 
 DESCRIPTION="Combination editor, browser, and mail client that is 100% text based"
 HOMEPAGE="https://edbrowse.org"
-SRC_URI="https://github.com/CMB/edbrowse/archive/v${PV}.tar.gz -> ${P}.tar.gz
+SRC_URI="https://github.com/CMB/edbrowse/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
```

Comment:
Please don't. This is extremely unlikely for most of the packages, and the URLs are becoming too long.

Reply:
(this is a general comment, not just for this one package)

Reply:
I agree with @mgorny: I don't really see the benefit of doing such thing

Reply:
Feel free to review / cherry-pick the other commits then :)